src/_guides/language/spec.md

```diff
@@ -9,26 +9,29 @@ For a gentler introduction to Dart, see the
 
 ## Dart 2
 
+### Latest completed specification
```

Comment:
Why do we need this section head? It seems unnecessary.

Reply:
Removed